components/centos/centos-openshift-setup/Vagrantfile

```diff
@@ -6,7 +6,7 @@ DOCKER_REGISTRY="docker.io"
 
 # The name of the OpenShift image.
```

Comment:
@praveenkumar can you please change the comment to make it explicit that we are talking about Docker hub tags here?